ui/shared/components/panel/variants/Variants.jsx

```diff
@@ -111,7 +112,11 @@ const Variant = React.memo(({ variant, isCompoundHet, mainGeneId, linkToSavedVar
         {isCompoundHet && <VariantIndividuals variant={variant} isCompoundHet />}
         {isCompoundHet && showReads}
       </Grid.Column>
-      <Grid.Column><Annotations variant={variant} /></Grid.Column>
+      <Grid.Column>
+        <Annotations variant={variant} />
+        <VerticalSpacer height={12} />
+        {variant.classification !== undefined && <Acmg variant={variant} />}
```

Comment:
This is not the criteria you should use to determine whether or not to show the ACMG classification. As we have said in countless meetings, it is shown on saved variants and not on unsaved variants. The way to determine whether or not a variant is saved or not is if it has the `variantGuid` property